README.md

```diff
@@ -1,32 +1,167 @@
 # Exploring the Likelihood of the Spoiler Effect in Ranked Choice Voting Systems
 
 ## Project Summary
+This project aims to quantify how common the Spoiler Effect is in Ranked Choice Voting systems. While this project 
+is still a work in progress, the end state is to have a web application that feeds user-generated ballots into a 
+machine learning model. This model will output probability of the Spoiler Effect occurring, based on the ballots 
+in the user-generated election. 
 
 ## Motivation
+In [2019](https://www.politico.com/states/new-york/albany/story/2019/11/05/ranked-choice-voting-adopted-in-new-york-city-along-with-other-ballot-measures-1226390), New York City (NYC) voted to start using a ranked choice system of 
+voting. One of this project's team members lived in NYC at the time and could vote in this election. In the days 
+leading up to the vote, she was hearing from both sides of the issue, and one argument stood out: that in a Ranked 
+Choice Voting (RCV) system, a candidate whom *no one* felt strongly about could win. 
+
+This phenomenon incorporates 
+various theorems (including, most famously, [Arrow's Impossibility Theorem](https://en.wikipedia.org/wiki/Arrow's_impossibility_theorem)), but generally goes by the name 'The Spoiler Effect.' She and this project's 
+other teammates wanted to know if this 'spoiler effect' was something voters should worry about, or if it 
+was so rare as to not affect the outcome of most elections.
+
+Of course, this project is still a work in progress years after the 2019 election. This is because a secondary 
+goal of this project is to simply increase our data science & software engineering skills. So, we have been taking our 
+time with each step in order to maximize learnings along the way (and, of course, we are doing this in our 'free' time). 
```

Comment:
Not sure if we need lines 20-22? Or we can simplify to say "we did this for fun on our free time".

Reply:
Okay yes, good idea :) Keep it short n' simple!